shared/src/main/java/org/odk/collect/shared/collections/CollectionExtensions.kt

```diff
@@ -8,4 +9,13 @@ object CollectionExtensions {
         val index = abs(any.hashCode()) % this.size
         return this[index]
     }
+
+    fun JSONArray.has(value: Any): Boolean {
```

Comment:
I think this should be `contains` instead to be more consistent with Java's `Collection`.

Reply:
In `JSON` world it's rather `has`, for example, JSONObject has `has` function so I wanted to make it consistent.